client/client_auth_test.go

```diff
@@ -88,13 +87,15 @@ func TestClientAuth(t *testing.T) {
 		http.Header{},
 	}
 
-	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	ts := startTestHTTPS(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		responder.headers = r.Header
 		w.WriteHeader(responder.httpStatus)
 		w.Header().Set("Content-Type", "application/json")
 
 		fmt.Fprint(w, responder.data)
-	}))
+	}),
+		localhostCert,
+		localhostKey)
```

Comment:
The indentation here is strange

Reply:
blame `go fmt` I can change it to your liking :) I will try.

Reply:
I prefer having a line with the same number of closing brackets on a single line as an opposing line introduces. IOW, if you add a newline after `startTestHTTPS(` the indentation should look OK.

Reply:
sure! thanks. looks really better.